packages/unplugin-vue-i18n/build.config.ts

```diff
@@ -24,5 +29,27 @@ export default defineBuildConfig({
   rollup: {
     emitCJS: true
   },
-  externals: ['vite', 'webpack']
+  externals: ['vite', 'webpack'],
+  hooks: {
+    'build:done': async () => {
+      await Promise.all([
+        rm(resolve(lib, 'types.cjs')),
+        rm(resolve(lib, 'types.mjs')),
+        ...['vite', 'webpack'].map(async name => {
```

Comment:
This is about removing the extra required `.default` when requiring `vite` or `webapack`  plugin (I don't use default export anymore to avoid this, you can remove the hack, run `pnpm build && pnpm pack` from the package and upload the generated tgz to https://arethetypeswrong.github.io/ or just check https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseExportDefault.md):
```ts
const webpack = require('@intlify/unplugin-vue-i18n/webpack').default.webpack
```